pom.xml

```diff
@@ -402,6 +402,13 @@
         </dependency>
 
         <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>dashboard-view</artifactId>
+            <version>2.9.2</version>
+            <optional>false</optional>
```

Comment:
Can't this dependency be optional just like build-pipeline-plugin or promoted-builds plugin?